src/api/user.js

```diff
@@ -0,0 +1,20 @@
+import { USER_OK } from 'test/mocks/user';
+import { makeRequest, METHODS } from 'api/apiManager';
+
+export const getUser = (params) => {
+  if (params) {
+    console.info(`calling API /user/${params}`);
+  }
+  return makeRequest({
+    uri: '/users/',
+    method: METHODS.GET,
+    mockResponse: USER_OK.payload,
```

Comment:
Please refactor USER_OK removing the payload layer, because it's already handled by makeRequest